manifests/t/Trados/TradosStudio2021/16.1.4276/Trados.TradosStudio2021.locale.en-US.yaml

```
@@ -1,18 +1,16 @@
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultlocale.1.0.0.schema.json
-PackageIdentifier: Rws.TradosStudio
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Trados.TradosStudio2021
 PackageVersion: 16.1.4276
 PackageLocale: en-US
-Publisher: Rws
+Publisher: SDL Limited (a part of the RWS Holdings Plc group)
```

```suggestion
Publisher: SDL
```
This must match Control Panel.

---

Matched to ARP